main.py

```diff
@@ -40,15 +42,40 @@ def delete_destination(destination_name):
     try:
         destination_delete = Destinations.query.filter_by(Name=destination_name).first()
         if destination_delete is None:
-            return jsonify({'error': 'Destination not found' }), 404
+            return jsonify({'error':'Destination not found' }), 404
 
         db.session.delete(destination_delete)
         db.session.commit()
-        return jsonify ({'message': 'Destination deleted Succesfully'}), 200
+        return jsonify ({'message':'Destination deleted Succesfully'}), 200
     except Exception as e:
         return jsonify({'error': str(e)}), 500
 
+@app.route('/destinations', methods=['PATCH'])
+def update_ideal_temp():
+    try:
+        destination_name = request.json.get('destination_name')
+        current_temperature = request.json.get('current_temperature')
+
+        if destination_name is None:
+            return jsonify({'error' : 'Destination name is required'}), 422
+        if current_temperature is None:
```

Comment:
You'll hate me.  But do you need to search on the current temperature as well?  I would assume we just need to get the destination by its name and then update it with its new temperature?  Or are you envisioning we have multiple temperatures per destination?  I would hope the latter.

So change this by only providing the destination_name and new_temperature in the request and update destination that way